app/views/promotion/termsPage.scala.html

```diff
@@ -30,7 +29,7 @@
             <h4>Applies to products</h4>
             <ul class="promotion-applies-to">
                 @catalog.allSubs.flatten.filter(plan => promotion.appliesTo.productRatePlanIds.contains(plan.id)).map { plan =>
-                    <li><a class="u-link" href="@routes.Checkout.renderCheckout(UK.id, None, None, plan.slug).url">@{plan.product.toString.replace("Digipack", "")} @plan.name</a></li>
+                    <li>@{plan.product.toString.replace("Digipack", "")} @plan.name</li>
```

Comment:
I removed the hyperlink here as it didn't have the promo code in and it may have confused users.